hexrdgui/llnl_import_tool_dialog.py

```diff
@@ -46,7 +46,7 @@ class LLNLImportToolDialog(QObject):
 
     cancel_workflow = Signal()
 
-    complete_workflow = Signal()
+    complete_workflow = Signal(bool)
```

Comment:
Can you leave a comment above this line as well that indicates what the `bool` is for?